walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -13,11 +13,118 @@
 // limitations under the License.
 
 package com.google.sps;
-
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+  public Collection <TimeRange > query(Collection <Event> events, MeetingRequest request) {
+    ArrayList <Event > eventsList = new ArrayList <>(events);
```

Comment:
No space before `>` for generics.

Reply:
`    ArrayList<Event> eventsList = new ArrayList<>(events);`

Reply:
ok!